MaterialDesignThemes.MahApps/Themes/MaterialDesignTheme.MahApps.RangeSlider.xaml

```diff
@@ -121,25 +124,25 @@
                         Background="Transparent"
                         Orientation="Horizontal">
                 <RepeatButton x:Name="PART_LeftEdge"
-                              Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=(controls:SliderHelper.TrackFillBrush)}"
+                              Foreground="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=Foreground}"
```

Comment:
What is the reason for changing away from the attached properties to Foreground? Typically this results in less customizability.

Reply:
Yes, not sure what I was thinking there.
Reverted.